resources/global/2022-2023/Annonce de rentrée.md

```diff
@@ -0,0 +1,84 @@
+Hello à toutes et à tous 👋 :)
+
+On espère que vous avez passé un bel été et une bonne rentrée pour celles et ceux qui ont déjà repris 
+
+:small_blue_diamond: **`Pour celles et ceux qui nous rejoignent`**
+
+Ef'Réussite est le serveur d'entraide étudiante pour les étudiants de l'Efrei.
+Géré par l'association Horizon, il permet ainsi à tout étudiant voulant donner de son temps pour aider ses camarades ou d'autres étudiants de sa promo qui auraient des questions ou des difficultés, en échange d'une reconnaissance pour sa participation dans cet objectif d'entraide... 
```

Comment:
```suggestion
Il est géré par l'association <@&805883190878928907>, par des étudiants bénévoles souhaitant aider leurs pairs (appelés <@&805873100813500447> ou eProfs). 

Nous proposons :
- des séances de révisions en ligne sur Ef'Réussite 💻
- des séances en présentiel sous la forme de cours ou de [mentorat en groupe](https://discord.gg/G7fWxQZXqF) 📚
- des enregistrements de ces séances stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) 🔗
- des ressources pédagogiques disponibles dans les canaux "-docs" 📄
- du [soutien pour vos projets de développement](https://discord.gg/TZ7VUHE9yf) et pour aller plus loin en programmation ! 🚀
```

Reply:
```suggestion
Il est géré par les l'association <@&805883190878928907>, par des étudiants bénévoles, appelés <@&805873100813500447> ou eProfs, souhaitant aider leurs pairs. 

Nous proposons :
- des séances de révisions en ligne sur Ef'Réussite 💻
- des séances en présentiel suivant la demande et le nombre d'étudiants intéressés
- des séances en présentiel de [mentorat en groupe assurées par le Pôle Mentorat](https://discord.gg/G7fWxQZXqF)
- des enregistrements de ces séances stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) 🔗
- des ressources pédagogiques disponibles dans les canaux "-docs" 📄
- du [soutien pour vos projets de développement](https://discord.gg/TZ7VUHE9yf) et pour aller plus loin en programmation ! 🚀
```
Suggestion bis but not taken into account - redoing whole announcement